config/crd/bases/config.tanzu.vmware.com_clientconfigs.yaml

```diff
@@ -85,16 +85,16 @@ spec:
                         description: LocalDiscovery is set if the plugins are to be
                           discovered via Local Manifest fast.
                         properties:
-                          manifestPath:
-                            description: ManifestPath is a local path pointing to
-                              manifest file
-                            type: string
                           name:
                             description: Name is a name of the discovery
                             type: string
+                          path:
+                            description: Path is a local path pointing to directory
+                              containing a list of CLIPlugin API resources
```

Comment:
**nitpick**: It seems imprecise to say that a directory contain a list. Typically, a directory contains files :smile:

I think this supposed to be something like: a directory containing YAML files, each of which contain one or more CLIPlugin API resources? (But I haven't finished reading the PR and there are some potentially important details to note here: can a given file contain multiple CLIPlugin API resources? are non-YAML files ignored? are other resources in the files ignored?)